### DISCUSSION TITLE
Show only matched Hits from search, not the entire list of hits.

### DISCUSSION THREAD

**Discussion by swachchhakh** [algolia/instantsearch#5309] [Q&A] [answered]
Hi, I am using react-instantsearch-native for my react-native app.
By default it shows all the hits. Even if i put a query inside the searchbox it shows me matched highlighted hits and also all the other hits.
It basically shows whole list of hits and then highlight the matched ones.
I want to show just the matched hits not the entire list.
![Screenshot_1650056209](https://user-images.githubusercontent.com/49757436/163631614-7d1581a8-8313-4b06-a222-194bd6afe313.png)

**Comment by Haroenv** [comment, marked as answer]
I think this is a confusion as that example only shows the title of the hit, even though it has other attributes as well, which explain why they are displayed in the UI. Hope that helps!
